File: UI/wwwroot/Facturacion/FrontModel/Lotes.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
class Lotes extends EntityClass {
   constructor(props) {
       super(props, 'EntityFacturacion');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Transaccion;
   /**@type {String}*/ Name;
   /**@type {String}*/ Descripcion;
   /**@type {Date}*/ Fecha;
   /**@type {Number}*/ Id_Usuario;
   /**@type {Number}*/ Id_Tipo_transaccion;
   /**@type {String}*/ Estado;
}
export { Lotes };

File: BusinessLogic/Facturacion/Mapping/Lotes.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using APPCORE;
namespace DataBaseModel {
   public class Lotes : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Transaccion { get; set; }
       public string? Descripcion { get; set; }
       public DateTime? Fecha { get; set; }
       public int? Id_Usuario { get; set; }
       public int? Id_Tipo_transaccion { get; set; }
       public string? Estado { get; set; }
   }
}
